test/test_bmark.jl

```diff
@@ -0,0 +1,75 @@
+using DataFrames
+using Logging
+using NLPModels
+using SolverTools
+
+# TODO: remove the SolverBenchmark. qualifiers below
+# when the corresponding methods have been removed
+# from SolverTools.
+
```

Comment:
```suggestion

```

Reply:
First, we need to update ST and release.

Reply:
Done